workspaces/util/src/__tests__/squashWhitespace-test.ts

```diff
@@ -0,0 +1,35 @@
+import { describe, expect, it } from "@jest/globals";
+
+import { squashWhitespace } from "../squashWhitespace";
+
+describe("squashWhitespace", () => {
+  it("removes whitespaces between words if it exists", () => {
```

Comment:
nit: from what I've seen, "whitespace" is a collective noun, like "bread", so we don't have to add S to make it plural

please update here and below
```suggestion
  it("removes whitespace between words if it exists", () => {
```